internal/pkg/aws/ecs/errors.go

```diff
@@ -0,0 +1,11 @@
+package ecs
+
+import "fmt"
+
+// ErrNoDefaultCluster occurs when the default cluster is not found.
+type ErrNoDefaultCluster struct {}
```

Comment:
can we have: `var ErrNoDefaultCluster = errors.New("default cluster does not exist")`